### ISSUE TITLE
release 2.0b7 broken

### ISSUE THREAD

**Issue by frisi** [Raptus/raptus.article.files#3]
the release https://pypi.python.org/pypi/raptus.article.files/2.0b7 is broken

the docs/ directory seems to be missing in the zip file

```
/home/zope/project/eggs/setuptools-12.4-py2.7.egg/pkg_resources/__init__.py:2512: PEP440Warning: 'distribute (0.6.24dev-r0)' is being parsed as a legacy, non PEP 440, version. You may find odd behavior and sort order. In particular it will be sorted as less than 0.0. It is recommend to migrate to PEP 440 compatible versions.
Processing raptus.article.files-2.0b7.zip
Writing /tmp/easy_install-AZY2yf/raptus.article.files-2.0b7/setup.cfg
Running raptus.article.files-2.0b7/setup.py -q bdist_egg --dist-dir /tmp/easy_install-AZY2yf/raptus.article.files-2.0b7/egg-dist-tmp-UPBe8C
error: docs/MANUAL.txt: No such file or directory
An error occurred when trying to install raptus.article.files 2.0b7. Look above this message for any errors that were output by easy_install.
While:
  Installing instance1.
  Getting distribution for 'raptus.article.files==2.0b7'.
Error: Couldn't install: raptus.article.files 2.0b7
```

**Comment by frisi** [comment]
@skaeser any ideay why the file was not been included in the release.
the MANIFEST.in looks okay to me

**Comment by skaeser** [comment]
Fixed in 2.0b8. (see ba1037d)

**Comment by frisi** [comment]
thanks simon
